### DISCUSSION TITLE
Calendar on Detail Page

### DISCUSSION THREAD

**Discussion by AkhilTreblle** [wdelfuego/nova-calendar#64] [Ideas & Feature Requests]
I am using this calendar to show individual employee attendance. Therefore, I would like to add this calendar to the employee resource detail page. Is that possible with the current update?

**Comment by wdelfuego** [comment]
It is not currently possible to add the calendar to detail pages as a [Resource Tool](https://nova.laravel.com/docs/4.0/customization/resource-tools.html), only to the Nova application as a regular [Tool](https://nova.laravel.com/docs/4.0/customization/tools.html).

I agree that it'd be valuable to be able to use it as a Resource Tool, so I'd like to look into that for a future update.

In the meantime, you could consider using [Event Filters](https://wdelfuego.github.io/nova-calendar/event-filters.html) to allow the end user to choose which employee to see the attendance for. The calendar wouldn't be shown on the Employee Detail page but be a separate view like now, but it'd be an easy way for your end users to at least be able to select the employee they want to see the data of.

I'd be happy to talk and think with you about the best way to go forward with this in your application.
